python/cudf/cudf/_lib/pylibcudf/interop.pyx

```diff
@@ -170,6 +208,31 @@ def to_arrow(cudf_object, metadata=None):
     raise TypeError("to_arrow only accepts Table and Scalar objects")
 
 
+@to_arrow.register(DataType)
+def _to_arrow_datatype(cudf_object, **kwargs):
```

Comment:
*nit*: Add documentation of the accepted `kwargs` and describe how they are used.

Reply:
thanks @wence- !